os_net_config/impl_nmstate.py

```diff
@@ -43,8 +43,10 @@
 import os_net_config
 from os_net_config import common
 from os_net_config import objects
+from os_net_config import sriov_config
```

Comment:
Don't import sriov_config here. The numvfs could be obtained from the iface_state() itself.